Content.Shared/Interfaces/GameObjects/Components/Interaction/IAfterInteract.cs

```diff
@@ -15,7 +16,7 @@ public interface IAfterInteract
         /// <summary>
         /// Called when we interact with nothing, or when we interact with an entity out of range that has no behavior
         /// </summary>
-        void AfterInteract(AfterInteractEventArgs eventArgs);
+        Task IAfterInteract(AfterInteractEventArgs eventArgs);
```

Comment:
You probably didn't intend to rename this method?

Reply:
I am not sure what the right call was there, I followed the refactoring tips.
It is a separate method I mistakenly renamed, right?

Reply:
Although the method was originally named IAfterInteract, wasn't it?

Reply:
no, the method was originially called `AfterInteract`. The **Interface** it is part of is called `IAfterInteract`.